Assets/_BForBoss/_Weapons/Scripts/ScriptableObjects/WeaponScriptableObject.cs

```diff
@@ -9,18 +9,21 @@ public interface IWeaponProperties
         float RateOfFire { get; }
         [CanBeNull] Sprite Crosshair { get; }
         float BulletSpread { get; }
+        int NumberOfBullets { get; }
     }
     
     [CreateAssetMenu(fileName = "WeaponProperties", menuName = "PerigonGames/Weapon", order = 1)]
     public class WeaponScriptableObject : ScriptableObject, IWeaponProperties
     {
         [SerializeField] private float _rateOfFire = 0.1f;
         [SerializeField] private float _bulletSpread = 1f;
+        [SerializeField] private int _numberOfBullets = 1;
         [PreviewField]
         [SerializeField] private Sprite _crosshairImage = null;
 
         public float RateOfFire => _rateOfFire;
-        [CanBeNull] public Sprite Crosshair => _crosshairImage;
         public float BulletSpread => _bulletSpread;
+        public int NumberOfBullets => _numberOfBullets;
+        [CanBeNull] public Sprite Crosshair => _crosshairImage;
```

Comment:
Since this is a [Jetbrains notation](https://www.jetbrains.com/help/resharper/Reference__Code_Annotation_Attributes.html), does this work for IDEs outside Rider? (Maybe Stupid Question)

Would a [nullable notation](https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/builtin-types/nullable-value-types) be better?